src/lib/mod.rs

```diff
@@ -36,7 +41,13 @@ impl ProtonClient {
     /// ```
     pub fn new(url: &str) -> Self {
         let client = Client::default().with_url(url.to_string());
+        let client_without_compression =
+            Client::with_compression(client.clone(), Compression::None).with_url(url.to_string());
```

Comment:
```suggestion
            Client::with_compression(client.clone(), Compression::None);
```

Since it's cloned from `client`, calling `with_url` should not be needed.